Tests/Cosmos.Compiler.Tests.Bcl/System/ByteTest.cs

```diff
@@ -44,6 +44,115 @@ public static void Execute()
 
             Assert.IsTrue((result == expectedResult), "Byte.ToString(X2) doesn't work");
 #endif
+
+            // basic bit operations
+
+            int val2;
+
+            value = 0x0C; // low-order bits: 0b0000_1100
+
+            val2 = ~value; // low-order bits: val2 = ~value = 0b1111_0011
+            Assert.IsTrue(val2 == -0x0D, "Byte bitwise not doesn't work got: " + val2);
+
+            val2 = value & 0x06; // low-order bits: val2 = value & 0b0000_0110 = 0b0000_0100
+            Assert.IsTrue(val2 == 0x04, "Byte bitwise and doesn't work got: " + val2);
+
+            val2 = value | 0x06; // low-order bits: val2 = value | 0b0000_0110 = 0b0000_1110
+            Assert.IsTrue(val2 == 0x0E, "Byte bitwise or doesn't work got: " + val2);
+
+            val2 = value ^ 0x06; // low-order bits: val2 = value ^ 0b0000_0110 = 0b0000_1010
+            Assert.IsTrue(val2 == 0x0A, "Byte bitwise xor doesn't work got: " + val2);
```

Comment:
I'm unsure if these test should be part of "Byte" while yes are Byte operations in C# are done all with integers indeed 'val2' is an int variable not a Byte.